server/src/test/java/com/lunchtime/controllers/FeedbackControllerTest.java

```diff
@@ -0,0 +1,84 @@
+package com.lunchtime.controllers;
+
+import com.lunchtime.service.dto.FeedbackDto;
+import com.lunchtime.service.impl.FeedbackServiceImpl;
+import com.lunchtime.models.Feedback;
+import com.lunchtime.models.Person;
+import com.lunchtime.models.Restaurant;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.MockitoAnnotations.initMocks;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+public class FeedbackControllerTest {
+
+    @Mock
+    private FeedbackServiceImpl mockFeedbackServiceImpl;
+
+    private FeedbackController feedbackControllerUnderTest;
+
+    @Before
+    public void setUp() {
+        initMocks(this);
+        feedbackControllerUnderTest = new FeedbackController(mockFeedbackServiceImpl);
+    }
+
+    @Test
+    public void testCreate() throws Exception {
```

Comment:
Test names should be self documented. Your colleagues should read test name and understand what tested method or class does

Reply:
Thanks!